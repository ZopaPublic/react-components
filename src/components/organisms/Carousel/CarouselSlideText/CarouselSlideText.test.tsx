import { render } from '@testing-library/react';
import axe from '../../../../../axe-helper';
import React from 'react';
import { Carousel } from '..';

describe('<Carousel.SlideText />', () => {
  it('renders with no a11y violations', async () => {
    const { container } = render(
      <Carousel.SlideText>We check your car and dealership so you can buy with confidence</Carousel.SlideText>,
    );
    const results = await axe(container.innerHTML);
    expect(results).toHaveNoViolations();
    expect(container).toMatchSnapshot();
  });
});
