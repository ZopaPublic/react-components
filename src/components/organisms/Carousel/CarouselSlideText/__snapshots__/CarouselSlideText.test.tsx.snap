// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Carousel.SlideText /> renders with no a11y violations 1`] = `
.c0 {
  margin: 0;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  color: #4A545E;
  font-size: 16px;
  line-height: 24px;
  font-weight: 400;
  font-family: "Open Sans",Roboto,Helvetica,Arial,sans-serif;
  text-align: inherit;
}

<div>
  <p
    class="c0"
    color="#4A545E"
  >
    We check your car and dealership so you can buy with confidence
  </p>
</div>
`;
